### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by tnyandoro**
In your React component responsible for removing books - implement that even by dispatching a corresponding action.

Use the redux-logger to check if your application is working correctly - Open your browser's console and look for the logger messages. Upon every action dispatch, the logger will print:

prev state - the state of the whole Redux store before the current action was dispatched.
action - the action object that is being dispatched.
next state - the state of the Redux store after the current action was dispatched.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -1,24 +1,53 @@
...
-      <button type="submit" onClick={() => dispatch(removeBook(id))}>Remove Book</button>
+      <h4>ADD A NEW BOOK</h4>
+      <div>
+        <ul>
+          { books && books.map((book) => (
+            <li key={book.id}>
+              <span>{book.title}</span>
+              <button id={book.id} type="button" onClick={removeBookFromStore}>Delete</button>
+            </li>
+          ))}
```
Comment (vichuge) [inline src/components/Books.js]: - [x] Good job implementing each book! Now, let's change it a little bit. In place to only HTML elements for each book, why not create a react component for each book? Here you can call it `<Book />` and then add all the info with more attributes. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (tnyandoro) [reply src/components/Books.js]: Created the book component

**Comment by Davidosky007** [comment]
## Hi @tnyandoro 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**